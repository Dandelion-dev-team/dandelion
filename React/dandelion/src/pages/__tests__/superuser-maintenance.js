import React from "react"
import renderer from "react-test-renderer"

import superUser from "../superuser-maintenance"

jest.mock("../../auth");
jest.mock("../superuser-maintenance");


describe("Superuser Maintenance", () => {
  it("renders correctly", () => {
    const tree = renderer
    .create(<superUser />)
    .toJSON()
  expect(tree).toMatchSnapshot()
  })
})