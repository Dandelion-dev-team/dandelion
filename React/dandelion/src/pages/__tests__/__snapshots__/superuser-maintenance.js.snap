// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Superuser Maintenance renders correctly 1`] = `<superUser />`;
